server/backend/courses/views.py

```
@@ -1,9 +1,10 @@
 from rest_framework import generics, viewsets
 from rest_framework.response import Response
-from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.decorators import permission_classes
```

Ruff (0.8.2)

`rest_framework.decorators.permission_classes` imported but unused

Remove unused import: `rest_framework.decorators.permission_classes`

(F401)